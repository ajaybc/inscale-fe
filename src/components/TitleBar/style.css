.TitleBar {
  background-color: #fff;
  box-shadow: 0.1px 1px 4px 0 #e7eeef;
  padding: 20px 0;
}

.TitleBar__title {
  margin: 0 0 10px 0;
  font-size: 1.6em;
  color: #0e141e;
}

.TitleBar__summary {
  color: #b7c9cc;
}