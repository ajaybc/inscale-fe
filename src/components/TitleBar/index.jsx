import React from 'react';
import Container from "../Container";

import './style.css';

export default ({title, summary}) => {
  return <div className="TitleBar">
    <Container>
      <h1 className="TitleBar__title">{title}</h1>
      <div className="TitleBar__summary">{summary}</div>
    </Container>
  </div>
} 