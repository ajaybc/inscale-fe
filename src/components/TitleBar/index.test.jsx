import React from 'react';
import renderer from 'react-test-renderer';

import TitleBar from './index.jsx';

it('Titlebar renders correctly when title and summary are provided', () => {
  const tree = renderer
    .create(<TitleBar title="Test Title" summary="Test Summary"/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});