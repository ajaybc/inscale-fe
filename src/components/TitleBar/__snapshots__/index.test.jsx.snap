// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titlebar renders correctly when title and summary are provided 1`] = `
<div
  className="TitleBar"
>
  <div
    className="Container"
  >
    <h1
      className="TitleBar__title"
    >
      Test Title
    </h1>
    <div
      className="TitleBar__summary"
    >
      Test Summary
    </div>
  </div>
</div>
`;
